src/analysis/classic/hunter/marksman/metrics/killCommandMaxCasts.ts

```diff
@@ -0,0 +1,56 @@
+import { AnyEvent, EventType } from 'parser/core/Events';
```

Comment:
this should probably get moved to `src/analysis/classic/hunter/shared` and then just imported in each of the specs that use it